helpers/progress.go

```diff
@@ -10,16 +10,30 @@ func (DiscardProgressWriter) Write(p []byte) (int, error) {
 	return len(p), nil
 }
 
-// UpdateTitle doesn't do anything but satisfy implementation
-func (DiscardProgressWriter) UpdateTitle(_ string) {}
+// Close doesn't do anything but satisfy implementation
+func (DiscardProgressWriter) Close() error {
+	return nil
+}
+
+// Updatef doesn't do anything but satisfy implementation
+func (DiscardProgressWriter) Updatef(_ string, _ ...any) {}
+
+// Successf doesn't do anything but satisfy implementation
+func (DiscardProgressWriter) Successf(_ string, _ ...any) {}
+
+// Failf doesn't do anything but satisfy implementation
+func (DiscardProgressWriter) Failf(_ string, _ ...any) {}
 
 // DiscardProgressWriter is a ProgressWriter in which all calls succeed without doing anything
 // Use this or nil or if you don't care about writing progress
 type DiscardProgressWriter struct{}
 
-// ProgressWriter wraps io.Writer, but also includes an updateTitle function to give the user
+// ProgressWriter wraps io.Writer, but also includes functions to give the user
 // additional context on what's going on. Useful in OCI for tracking layers
 type ProgressWriter interface {
-	UpdateTitle(string)
+	Updatef(string, ...any)
```

Comment:
If we are already going to break the interface, I think `Printf` works better here

Reply:
IMO print implies it would be a running log which isn't accurate for most of the progress/spinner interfaces we have (they mostly replace / update what is already printed)